src/sql/platform/bootstrap/browser/bootstrapService.ts

```diff
@@ -39,15 +40,21 @@ function createUniqueSelector(selector: string): string {
 let platform: PlatformRef;
 
 export function bootstrapAngular<T>(service: IInstantiationService, moduleType: IModuleFactory<T>, container: HTMLElement, selectorString: string, params: IBootstrapParams, input?: IEditorInput, callbackSetModule?: (value: NgModuleRef<T>) => void): string {
+	if (!platform) {
+		service.invokeFunction((accessor) => {
+			const environmentService = accessor.get(IEnvironmentService);
+			if (environmentService.isBuilt) {
```

Comment:
This looks backwards

Reply:
What do you mean by backwards? The boolean logic or?

Reply:
Just realized an issue here..

Reply:
There was a race condition here that should be solved now